### DISCUSSION TITLE
ghc-vis

### DISCUSSION THREAD

**Discussion by sorki** [haskell-nix/hnix#895] [General]
Yesterday someone on IRC posted https://www.youtube.com/watch?v=mDzg-e3OSz8 which uses [ghc-vis](http://felsin9.de/nnis/ghc-vis/) and it looks like it could be quite useful for us. Another video at http://felsin9.de/nnis/ghc-vis/#combined-debugger

Anyone tried this / using this already?

**Comment by Anton-Latukha** [comment]
Here we also talked about it half a year ago, when we talked about visualization tooling and debuggers.

As said, wanted to learn to use this thing. And played a little then.

But so far was working without using it, so far do not intuitively know the scopes of application for it.

**Comment by nomeata** [comment]
@dalpd seems to be trying to fix building `ghc-vis` on NixOS: https://github.com/NixOS/nixpkgs/compare/master...dalpd:fix-ghc-vis
